code_schemes/ws_correct_dataset.json

```diff
@@ -0,0 +1,105 @@
+{
+    "SchemeID": "Scheme-b47005f9",
+    "Name": "WS - Correct Dataset",
+    "Version": "0.0.0.1",
+    "Codes": [
+      {
+        "CodeID": "code-45133562",
+        "CodeType": "Normal",
+        "DisplayText": "age",
+        "StringValue": "age",
+        "NumericValue": 1,
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-154b6cb7",
+        "CodeType": "Normal",
+        "DisplayText": "gender",
+        "StringValue": "gender",
+        "NumericValue": 2,
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-7614a4d3",
+        "CodeType": "Normal",
+        "DisplayText": "constituency",
+        "StringValue": "constituency",
+        "NumericValue": 3,
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-5c2c99ab",
+        "CodeType": "Normal",
+        "DisplayText": "livelihood",
```

Comment:
Are all of the schemes above mentioned in this one?

Reply:
Also the display texts here don't match the scheme names you used earlier. If these display texts are enough, then why not use these as the scheme names?

Reply:
Let me standardise these thanks!